modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -4829,9 +4829,9 @@ protected String getCollectionFormat(Parameter parameter) {
         } else if (Parameter.StyleEnum.SIMPLE.equals(parameter.getStyle())) {
             return "csv";
         } else if (Parameter.StyleEnum.PIPEDELIMITED.equals(parameter.getStyle())) {
-            return "pipe";
```

Comment:
Have you verified that this isn't converting to `pipeDelimited` whenever for OpenAPI 3.0 support. Same for `spaceDelimited` below.

Reply:
for a definition with `openapi: 3.0.0` and `style: pipeDelimited` the emitted code is still using 
```
export const COLLECTION_FORMATS = {
    csv: ",",
    ssv: " ",
    tsv: "\t",
    pipes: "|",
};
```
in `base.ts` and `localVarQueryParameter['typeQueryParam'] = typeQueryParam.join(COLLECTION_FORMATS.pipes);` in the controller.